specification/storage/resource-manager/readme.python.md

```diff
@@ -2,10 +2,11 @@
 
 These settings apply only when `--python` is specified on the command line.
 
-``` yaml
+``` yaml $(python) && $(track2)
 azure-arm: true
 license-header: MICROSOFT_MIT_NO_VERSION
 package-name: azure-mgmt-storage
+package-version: 1.0.0b1
```

Comment:
@msyyc does this change required?

Reply:
Just needed by autorest tool. It does not influence python SDK code.